src/com/projectkorra/projectkorra/airbending/combo/AirStream.java

```diff
@@ -165,7 +166,7 @@ public void run() {
 				continue;
 			}
 			final Vector force = GeneralMethods.getDirection(entity.getLocation(), this.currentLoc);
-			entity.setVelocity(force.clone().normalize().multiply(this.speed));
+			GeneralMethods.setVelocity(this,entity, force.clone().normalize().multiply(this.speed));
```

Comment:
Please change to `GeneralMethods.setVelocity(this, entity, force.clone().normalize().multiply(this.speed));`